manager/logbroker/subscription.go

```diff
@@ -182,6 +183,11 @@ func (s *subscription) match() {
 				continue
 			}
 			for _, task := range tasks {
+				// if we're not following, don't add tasks that aren't running yet
+				if !s.follow() && task.Status.State < api.TaskStateRunning {
+					s.errors = append(s.errors, errors.Errorf("task %v is not yet running", task.ID))
```

Comment:
What do you think about https://github.com/docker/swarmkit/pull/1995/files#diff-799702dee23d082d1429bb199615ffe3R135 ?

I don't disagree but I'm following the pattern.